libs/brig-types/src/Brig/Types/Intra.hs

```diff
@@ -119,8 +119,9 @@ instance FromJSON UserAccount where
 
 instance ToJSON UserAccount where
   toJSON (UserAccount u s) =
-    let Object o = toJSON u
-     in Object $ M.insert "status" (toJSON s) o
+    case toJSON u of
+      Object o -> Object $ M.insert "status" (toJSON s) o
+      other -> other -- shouldn't happen
```

Comment:
probably better to fail here, since this is more likely to get caught in integration tests.

Reply:
done